pages/index.tsx

```diff
@@ -1,72 +1,89 @@
-import type { NextPage } from 'next'
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import type { NextPage } from "next";
+import Head from "next/head";
+import React, { useState } from "react";
+import styles from "../styles/Home.module.css";
 
 const Home: NextPage = () => {
+  type inputType = string;
+  type todoListType = Array<String>;
+
+  const [input, setInput] = useState<inputType>("");
+  const [todoList, setTodoList] = useState<todoListType>([]);
```

Comment:
```suggestion
  const [input, setInput] = useState<string>("");
  const [todoList, setTodoList] = useState<string[]>([]);
```
Không cần phải tạo ra 1 biến riêng đâu em, ngoài ra nhớ lưu vào cả localStorage nhé